pkgs/development/tools/mysql-shell/default.nix

```diff
@@ -0,0 +1,174 @@
+{ lib
+, stdenv
+, pkg-config
+, cmake
+, fetchurl
+, git
+, bison
+, openssl
+, protobuf
+, curl
+, zlib
+, libssh
+, zstd
+, lz4
+, boost
+, readline
+, libtirpc
+, rpcsvc-proto
+, libedit
+, libevent
+, icu
+, re2
+, ncurses
+, libfido2
+, v8
+, python
+, cyrus_sasl
+, openldap
+, numactl
+, cctools
+, CoreServices
+, developer_cmds
+, DarwinTools
+, testVersion
+, mysql-shell
+}:
+
+with builtins;
+let
+  pythonDeps = [
+    python.pkgs.certifi
+    python.pkgs.paramiko
+  ];
+  site = ''
+
+    import sys; sys.path.extend([${concatStringsSep ", " (map (x: ''"${x}/${python.sitePackages}"'') pythonDeps)}])
```

Comment:
Why are we not using the standard python tooling?

Reply:
Mysql Shell embeds the python runtime. It's not a typical python package. Any suggestions ?